sympy/integrals/integrals.py

```diff
@@ -1286,12 +1286,12 @@ def principal_value(self, **kwargs):
         if a == b:
             return 0
         r = Dummy('r')
+        f = self.function
         singularities_list = [s for s in singularities(self.function, x) if s.is_comparable and a <= s <= b]
```

Comment:
This line would now be shorter with `f` instead of `self.function`. In addition, part of it could be moved to the next line.

Reply:
>This line would now be shorter with `f` instead of `self.function`

done this..
>In addition, part of it could be moved to the next line.

What exactly are you talking about here?